src/transliteration/ui/WordInfo.tsx

```diff
@@ -54,6 +54,7 @@ export default function WordInfoWithPopover({
           {isReconstructionWord && (
             <Alignments
               tokenIndex={word.sentenceIndex}
+              lemma={word.uniqueLemma}
```

Comment:
We need to pass the lemma to check if it's the same as the reconstruction word's lemma.